testsuite/features/srv_monitoring.feature

```diff
@@ -4,14 +4,27 @@
 
 Feature: Monitor SUSE Manager server
 
+  Scenario: Disable monitoring from the UI
```

Comment:
What about adding a comment here which explains that because of sumaform we expect monitoring is "on"?

Reply:
Thanks, added a comment.